example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -483,17 +486,18 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
+				CODE_SIGN_ENTITLEMENTS = Runner/Runner.entitlements;
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = "$(FLUTTER_BUILD_NUMBER)";
-				DEVELOPMENT_TEAM = 8734Y6QF3F;
+				DEVELOPMENT_TEAM = 87RQQPQ85J;
 				ENABLE_BITCODE = NO;
 				INFOPLIST_FILE = Runner/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = flutter.stripe.example;
+				PRODUCT_BUNDLE_IDENTIFIER = "flutter.stripe.example-2";
```

Comment:
why do we need a new bundle identifier we should be able to add Apple Pay later to it right?

Reply:
I register that bundle identifier with my personal account where I longer don't pay the Developer fee therefore it was not possible to use ApplePay with that identifier